Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.14
+FROM golang:latest
```

Comment:
I would like also this one changed. Can be done in new PR

Reply:
Noted